src/features/commands/watch.js

```diff
@@ -0,0 +1,38 @@
+const { SlashCommandBuilder } = require('@discordjs/builders'),
+      db = require('../../db');
+
+module.exports = {
+    type: 'slash',
+    data: new SlashCommandBuilder()
+        .setName('watch')
+        .setDescription('Sends a direct message whenever keyword that you track gets mentioned.')
+        .addStringOption(option =>
+            option.setName('keyword')
+                .setDescription('The keyword you track.')
+                .setRequired(true)
+        ),
+    async execute (client, interaction) {
+        const keyword = interaction.options.getString("keyword");
+        const server = interaction.guild;
+        let watchedKeywordsCollection = db.getWatchedKeywords();
```

Comment:
```suggestion
        if (!server) {
          //Tell the user that this command can only be run in guilds and return
        }
        
        
        let watchedKeywordsCollection = db.getWatchedKeywords();
```

Reply:
Don't Dotsimus commands only work in guilds and not DMs or elsewhere?

Reply:
> Don't Dotsimus commands only work in guilds and not DMs or elsewhere?

Watch command only works within guilds and that's what @BeakerThe1st is trying to say, you have to make it throw an explanation that user shouldn't be trying to use them within DMs.

Reply:
> > Don't Dotsimus commands only work in guilds and not DMs or elsewhere?
> 
> Watch command only works within guilds and that's what @BeakerThe1st is trying to say, you have to make it throw an explanation that user shouldn't be trying to use them within DMs.

Well, I didn't know that slash commands could be ran though DMs so I thought it wasn't much of a problem, ok so I added the check.